control/passivity.py

```diff
@@ -0,0 +1,72 @@
+'''
+Author: Mark Yeatman  
+Date: May 15, 2022
+'''
+
+import numpy as np
+
+try:
+    import cvxopt as cvx
+except ImportError as e:
+    cvx = None
+
+lmi_epsilon = 1e-12
+
+def is_passive(sys):
```

Comment:
We have `(sys.)isdtime()`, `(sys.)isctime()`, and `(sys.)issiso()`. If we want to maintain a sense of consistency, this should be changed to `ispassive(sys)` and wrapped by a `sys.ispassive()` method for `StateSpace` systems, possibly even a wrapper for other LTI systems which realize them into `StateSpace` before calling the function in the passivity module.

`ispassive` would also be consistent with the naming scheme from [NumPy](https://numpy.org/doc/stable/reference/routines.logic.html) and [stdlib Python](https://docs.python.org/3/library/math.html#math.isclose)

Reply:
Renaming the function for consistency sounds good.

I am planning on trying to add some sort of passivity related tools for systems beyond ones that are LTI (an LMI to check passivity was the lowest hanging fruit). In [Demonstrating Passivity and Dissipativity using Computational Methods](https://www3.nd.edu/~pantsakl/Publications/497-isis08%202013.pdf), there's methods for using sum-of-squares to compute passivity for polynomial nonlinear systems and I believe there's literature elsewhere for systems described by combinations of polynomials and trig functions. 

I think it is a good suggestion, but I'm not immediately sure what parent class to try to add a wrapper for ispassive() to.